### PR TITLE
Data buckets tests created

### PR DESCRIPTION
**Pull request by mcada**
https://github.com/syndesisio/syndesis-qe/issues/288
For now it only tests popups. Next scenario will be on usage.

### CODE REVIEW

**Comment by mcada** [comment]
Added test for data buckets availability and usage from data mapper step

**Comment by mcada** [comment]
Development is done, test fails because of this issue: https://github.com/syndesisio/syndesis/issues/1701

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationFlowViewComponent.java [hunk 1]
```diff
@@ -99,15 +102,39 @@ public void clickRandomTrash() {
...
+     * @return text in popover element
+     */
+    public String checkTextInHoverTable(String stepPosition) {
+        String text;
+        if (stepPosition.equalsIgnoreCase("middle")) {
+            // This is ugly but for now it works - it has only one usage: for getting our data-mapper step
+            // which is between start and finish step.
+            // Explanation: we have 3 steps and between them 2 elements with insert step option --> 5 total
+            // with class "step" and we want the third element
+            $$(By.className("step")).shouldHaveSize(5).get(2)
```
Comment (dsimansk) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationFlowViewComponent.java]: I been wondering if those 2 numbers, `5` and `2` shouldn't be rather parameters of method?

Comment (dsimansk) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationFlowViewComponent.java]: Just to make it a bit less ugly :)

**Pull request merged**